test/lib/main.spec.js

```diff
@@ -87,6 +87,12 @@ describe('main', () => {
     expect(main.__get__('process').env.NODE_TLS_REJECT_UNAUTHORIZED).to.eq('0');
   });
 
+  it('supports actions that do not require an instance', async () => {
```

Comment:
Do we test the case that the instance data is still required for other actions? The "'Failed to obtain a url to the API'" case?

Reply:
I presumed we did already (since I didn't add that code flow), though I didn't look that hard. I'll take another look and add it if it's not there.

Reply:
i think i checked before leaving the comment and it wasn't present